### ISSUE TITLE
Width/Height

### ISSUE THREAD

**Issue by TannerJ07** [rlj1202/go-BitmapLogicSimulator#2]
Hello, it has probably been a while since you check out the code but is there a way to start the zoom level much lower? It gets annoying when working with small circuits.

**Comment by rlj1202** [comment]
You're right. It has been so long time ago. But as I remember, modifying this part of the code should solve your needs.

https://github.com/rlj1202/go-BitmapLogicSimulator/blob/061cd0939fbe5b74c2f7cbef5a849185acdc8587/cmd/BitmapLogicSimulator/main.go#L432

It limits the zoom level.
